rocket-chat-android-widgets/src/main/res/layout/message_inline_attachment.xml

```diff
@@ -93,14 +96,12 @@
 
         </LinearLayout>
 
-        <ImageView
+        <com.facebook.drawee.view.SimpleDraweeView
                 android:id="@+id/image"
-                android:layout_width="wrap_content"
-                android:layout_height="wrap_content"
-                android:maxHeight="200dp"
+                android:layout_width="match_parent"
+                android:layout_height="200dp"
```

Comment:
I tried it here: The scroll is bad.. really bad. We could improve that with an initial smaller size. Will try that.

Tha SVG? Won't show. We'll need some special code to deal with it.